vulture_os/services/frontend/models.py

```diff
@@ -2018,6 +2104,35 @@ def render_pre_ruleset(self):
                 logger.info(f"[RENDER PRE RULESET FAILURE] {log_om}")
         return result
 
+    def render_ruleset_options(self):
+        """ Render ruleset's options
+        :return  Str containing the rendered config
+        """
+        options_dict = {
+            "type": self.queue_type,
+            "size": self.queue_size,
+            "dequeueBatchSize": self.dequeue_batch_size,
+            "workerThreads": self.nb_workers,
+            "workerThreadMinimumMessages": self.new_worker_minimum_messages,
+            "timeoutshutdown": self.shutdown_timeout
+        }
+        if self.enable_disk_assist:
+            options_dict.update({
+                "highWatermark": self.high_watermark,
+                "lowWatermark": self.low_watermark,
+                "spoolDirectory": self.spool_directory or "/var/tmp",
+                "maxFileSize": f"{self.max_file_size}m" if self.max_file_size else None,
+                "maxDiskSpace": f"{self.max_disk_space}m" if self.max_disk_space else None,
+                "saveOnShutdown": "on" if self.save_on_shutdown else None,
+                "filename": f"{self.get_ruleset()}_disk-queue",
+                "checkpointInterval": 1024
+            })
+
+        options_dict = dict(filter(lambda x: x[1], options_dict.items()))
```

Comment:
prefer adding the elements explicitely with conditions, instead of using a less understandable lambda filter to filter-out None-elements